sovtokenfees/sovtokenfees/req_handlers/write_handlers/xfer_fee_handler.py

```diff
@@ -0,0 +1,13 @@
+from sovtoken.constants import XFER_PUBLIC
+
+from indy_common.authorize.auth_actions import AuthActionAdd
+from sovtoken.request_handlers.write_request_handler.xfer_handler import XferHandler
+
+from plenum.common.request import Request
+
+
+class XferFeeHandler(XferHandler):
```

Comment:
It would be great to add code initializing all these new handlers (in `main.py`). The code can be commented out.
I think it's useful to do this in order to understand if we have everything we need in a general Pluggable req handlers framework.

Reply:
ok